js/hotgraphicModel.js

```diff
@@ -10,10 +11,22 @@ export default class HotgraphicModel extends ItemsComponentModel {
 
   setUpItems() {
     super.setUpItems();
+    const id = this.get('_id');
+    this.getChildren().forEach((child, index) => {
 
-    this.getChildren().forEach((child) => {
       // Set _pin for the item if undefined
       if (!child.get('_pin')) child.set('_pin', false);
+
+      const tooltip = child.get('_tooltip');
+      if (!tooltip?._isEnabled) return;
+      tooltip._id = `hotgraphic-pin-${id}-${index}`;
+      tooltips.register({
+        ...child.toJSON(),
+        _classes: [ 'hotgraphic__pin-tooltip' ],
+        _position: 'outside bottom middle middle',
```

Comment:
Unsure if this is intentional or not but I thought I'd mention it just in case:

If there is no `_position` property is added to the hotgraphic item the position of the tooltip displays as follows:
<img width="241" alt="Screenshot 2023-04-27 at 11 36 01" src="https://user-images.githubusercontent.com/11569678/234838828-c50f2815-7302-4e8d-a60a-eb08c0b3ff1c.png">

If `_position` property is added but left empty the position of the tooltip displays as follows:
<img width="271" alt="Screenshot 2023-04-27 at 11 36 10" src="https://user-images.githubusercontent.com/11569678/234839994-c90a8b31-3a4a-453a-aec9-dd58218f8f47.png">

Reply:
Should be fixed. Hopefully. It'll default to `"'outside bottom middle middle"` if falsey now.

Reply:
Lovely, thank you